File: benchmarks/suite5/fib.rs
mod bench_runtime;
use bench_runtime::suite5_print_i64;

fn main() {
    let n: i64 = 5_000_000;
    let mut a: i64 = 0;
    let mut b: i64 = 1;
    let mut i: i64 = 0;
    while i < n {
        let c = a + b;
        a = b;
        b = c;
        i += 1;
    }
    suite5_print_i64(a);
}

File: benchmarks/suite5/fib.go
package main


func main() {
	n := suite5OpaqueI64(5000000)
	var a, b int64 = 0, 1
	var i int64
	for i = 0; i < n; i++ {
		c := a + b
		a = b
		b = c
	}
	suite5PrintI64(a)
}
